course_content/notebooks/cartopy_intro.ipynb

```diff
@@ -172,8 +172,8 @@
     "import matplotlib.pyplot as plt\n",
     "import cartopy.crs as ccrs\n",
     "\n",
-    "x = np.linspace(310, 390, 25)\n",
-    "y = np.linspace(-24, 25, 35)\n",
+    "x = np.linspace(337, 377, 25)\n",
+    "y = np.linspace(-18.7, 25.3, 35)\n",
```

Comment:
No, it's much higher than this (and I suspect the NAE was better than 25 * 35)

If you want the full blown thing it would be:
```
x = np.linspace(337, 377, 1000)
y = np.linspace(-18.7, 25.3, 1100)
```
which is rather large for the demo purposes, so had just kept it to the lat/long box of the domain rather than full (model level) resolution. Do you want me to up the resolution?